libenv/unix_iface.c

```diff
@@ -677,6 +675,45 @@ static void FindV6InterfacesInfo(EvalContext *ctx, Rlist **interfaces, Rlist **h
             }
         }
 
+        if (strcasestr(ifconfig_line, "ether"))
```

Comment:
**Better:**

```
if (strcasestr(ifconfig_line, "ether") != NULL)
```

**Best:**

```
if (StringStartsWith(TrimWhitespace(ifconfig_line), "ether"))
```

Reply:
We could add `StringStartsWithIgnoreCase()`. Have you seen `ether` uppercased somewhere?

Reply:
Nope, but why not :+1: